src/plugins/kibana/public/management/index.js

```diff
@@ -42,6 +42,16 @@ uiModules
       if ($scope.section) {
         $scope.section.items.forEach(item => {
           item.class = `#${$location.path()}`.indexOf(item.url) > -1 ? 'active' : undefined;
+          // Disable link if path is not set
+          if (!item.path) {
+            delete item.url;
+          }
+          item.items.forEach(childItem => {
+            // Disable link if path is not set
+            if (!childItem.path) {
+              delete childItem.url;
```

Comment:
I like how the property assignments for all the `options` happen via [this assign statement](https://github.com/elastic/kibana/blob/master/src/ui/public/management/section.js#L29). I think we should let `this.url` be populated the same way as well. That way, all `options` assignments to `this` properties are taking place in one spot in the code.

What I could do is define a helper function like `setUrl` that will take in the `options` object as a parameter and performs the aforementioned logic within it. Then I'd call that helper function from the constructor before the `assign` statement.